spec/core_functions/meta/get_mixin/content.hrx

```diff
@@ -0,0 +1,172 @@
+<===> passes-content/input.scss
+@use "sass:meta";
+
+@mixin foo {
+    @content;
+}
+
+a {
+    @include meta.apply(meta.get-mixin("foo")) {
+        b: red;
+    }
+}
+
+<===> passes-content/output.css
+a {
+  b: red;
+}
+
+<===>
+================================================================================
+<===> passes-empty-content/input.scss
+@use "sass:meta";
+
+@mixin foo {
+    @content;
+}
+
+a {
+    @include meta.apply(meta.get-mixin("foo")) {}
+}
+
+<===> passes-empty-content/output.css
+
+<===>
+================================================================================
+<===> denies-content/user-defined/input.scss
+@use "sass:meta";
+@mixin foo {}
+
+a {
+    @include meta.apply(meta.get-mixin("foo")) {}
+}
+
+<===> denies-content/user-defined/error
+Error: Mixin doesn't accept a content block.
+    ,
+2   | @mixin foo {}
+    |        === declaration
+... |
+5   |     @include meta.apply(meta.get-mixin("foo")) {}
+    |     ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ invocation
+    '
+  input.scss 5:5  root stylesheet
+
+<===>
+================================================================================
+<===> denies-content/builtin/input.scss
+@use "sass:meta";
+
+a {
+    @include meta.apply(meta.get-mixin(load-css, meta), "foo") {
+        a: b;
+    }
+}
+
+<===> denies-content/builtin/error
+Error: Mixin doesn't accept a content block.
+  ,--> input.scss
+4 |     @include meta.apply(meta.get-mixin(load-css, meta), "foo") {
+  |     ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ invocation
+  '
+  ,--> sass:meta
+1 | @mixin load-css($url, $with: null) {
+  |        =========================== declaration
+  '
+  input.scss 4:5  root stylesheet
+
+<===>
+================================================================================
+<===> scope/redeclare-vars/input.scss
```

Comment:
This should probably be nested in the `redeclare/` directory you define below instead of being a sibling directory.